Tasks/Common/webdeployment-common-v2/packageUtility.ts

```diff
@@ -23,6 +23,27 @@ export class PackageUtility {
 
         return availablePackages[0];
     }
+
+    public static getArtifactAlias(packagePath: string): string {
+        let artifactAlias = null;
+        let workingDirectory = tl.getVariable("system.defaultworkingdirectory");
+        try {
```

Comment:
Can you confirm your assumptions about the alias from someone in RM to handle any corner cases related to alias

Reply:
It is true that artifacts will be downloaded under defaultWorkDir\alias. However user can give any path in the input so we handle the case where we incorrectly determined the alias by doing tl.getVariable(`release.artifacts.${artifactAlias}.definitionId`. if this value is non-null then it means alias is correct else alias is null and all primary variables will be used